apps/sms/test/marionette/lib/views/thread.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var View = require('./view');
+
+function Thread() {
+  View.apply(this, arguments);
+}
+module.exports = Thread;
+
+Thread.prototype = {
+  __proto__: View.prototype,
+
+  callContact: function() {
+    this.pageAccessors.Thread.callButton.tap();
+
+    var Dialer = require('./dialer');
```

Comment:
As this is linked to the initialization of subpages, let's move this discussion on general thread.